types/sinon-chrome/index.d.ts

```diff
@@ -353,6 +356,23 @@ declare namespace SinonChrome.permissions {
     export var request: SinonChromeStub;
 }
 
+declare namespace SinonChrome.plugins {
+    export interface Plugin {}
+    export interface Translation {
+        message: string;
+        placeholders?: object;
+    }
+    export interface Translations {
+        [key: string]: Translation;
+    }
+    export class I18nPlugin implements Plugin {
+        constructor(translations: Translations);
```

Comment:
Looks like this parameter is actually optional too

Reply:
Thanks for the feedback.
I make it optional.